docs/DeveloperGuide.md

```diff
@@ -645,8 +645,81 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+### Adding a hairdresser
+
+1. Adding a hairdresser to **HairStyleX**
+
+   1. Prerequisites: Arguments are valid and compulsory parameters are provided. No duplicate hairdresser in the list.
+
+   1. Test case: `add_hairdresser n/Mary Ng p/98123732 e/maryn@example.com g/F ti/Senior Stylist s/Perm` <br>
+      Expected: Adds an hairdresser with a unique hairdresser ID, with name `Mary Ng`, phone `98123732`, email `maryn@example.com`, gender `F`, title `Senior Stylist`, specialisation `Perm`. 
+
+   1. Test case: `add_hairdresser n/Serena Goh`<br>
+      Expected: No hairdresser is added. Error details shown in result display.
+
+   1. Other incorrect add commands to try: `add_hairdresser`, `add_hairdresser s/wash`, `...`<br>
+      Expected: Similar to previous test case.
+
+### Editing a hairdresser
+
+1. Editing a hairdresser in **HairStyleX**
+
+   1. Prerequisites: Arguments are valid and compulsory parameters are provided. The hairdresser to be edited must be in **HairStyleX**. A hairdresser with hairdresser ID 1 must be present in the list. 
+
+   1. Test case: `edit_hairdresser 1 ti/Apprentice`<br>
+      Expected: Edits the hairdresser with ID of 1 in the displayed hairdresser panel with title `Apprentice`.
+
+   1. Test case: `edit_hairdresser ti/Apprentice`<br>
+      Expected: No hairdresser is edited. Error details shown in result display.
+
+   1. Other incorrect edit commands to try: `edit_hairdresser`, `edit_hairdresser x s/COMPLETED` (where there are no hairdressers with hairdresser ID of x), `...`<br>
```

Comment:
incorrect edit commands also because edit_hairdresser does not take in s/ prefix

Reply:
maybe change it to `edit_hairdresser x ti/Apprentice`